Assets/Code/Classes/Agent Hierarchy/Market.cs

```diff
@@ -15,6 +15,10 @@ public class Market : Agent
     public static int STARTING_ORE_AMOUNT = 0;
     public static int STARTING_ROBOTICON_AMOUNT = 12;
 
+    public static int TARGET_FOOD_AMOUNT = 16;
```

Comment:
Needs documentation. what are these for? also youve written them in constant format but not labelled them constant